$NetBSD: patch-aa,v 1.3 2007/03/17 14:19:59 rillig Exp $

gcc4 complains:
term.c:98: error: conflicting types for 'tparm'
term.c:98: note: a parameter list with an ellipsis can't match an empty parameter name list declaration
./../include/compat.h:10: error: previous declaration of 'tparm' was here

--- source/term.c.orig	2003-12-03 23:17:40.000000000 +0100
+++ source/term.c	2007-03-17 15:16:22.000000000 +0100
@@ -94,8 +94,7 @@ extern	int		tgetflag();
 #define Tgetflag(x) 	tgetflag(x.tname)
 #endif
 
-extern	char	*getenv();
-extern	char	*tparm();
+#include <stdlib.h>
 
 /*
  * The old code assumed termcap. termcap is almost always present, but on
